host/dynamicconfig.go

```diff
@@ -41,6 +41,7 @@ var (
 	staticOverrides = map[dynamicconfig.Key]any{
 		dynamicconfig.FrontendRPS:                                   3000,
 		dynamicconfig.FrontendESIndexMaxResultWindow:                defaultTestValueOfESIndexMaxResultWindow,
+		dynamicconfig.FrontendMaxNamespaceRPSPerInstance:            50,
```

Comment:
Not sure if `FrontendMaxNamespaceVisibilityBurstPerInstance` needs to be overwritten as well. 
I need to increase both on my local for the tests to pass.

Reply:
Set the visibility burst to 50.